src/common/uppy.utils.tsx

```diff
@@ -9,4 +14,131 @@ export const StyledUppyBox = styled(Box)(({ theme }) => ({
     width: 'calc(100% - 64px)',
   },
   '& .uppy-Dashboard--modal .uppy-Dashboard-inner': { zIndex: 1300 },
+  // The theme variable doesn't update as expected in the renderFields function.
+  // Therefore, all CSS that depends on the theme mode is handled here.
+  // The input focus styles are adapted from Uppy core styles:
+  // https://github.com/transloadit/uppy/blob/3aa8d688ff4a94b52287dbf737eb5b3f10b29dda/packages/%40uppy/core/src/_common.scss#L49
+  '.custom-div-focus': {
```

Comment:
The CSS class name custom-div-focus is too generic and could lead to potential naming clashes. To make it more descriptive and context-specific, consider including both ims and uppy in the class name, along with its purpose (focus state within an input).